message/messaging_service.hh

```diff
@@ -186,7 +186,9 @@ enum class messaging_verb : int32_t {
     TABLET_CLEANUP = 67,
     JOIN_NODE_REQUEST = 68,
     JOIN_NODE_RESPONSE = 69,
-    LAST = 70,
+    TABLET_STREAM_FILES = 70,
+    STREAM_BLOB = 71,
```

Comment:
Can we have also minimal unit tests for those?

Reply:
The implementation for those verbs and the unit tests go to enterprise branch.

Reply:
That's why you want synthetic unit tests in OSS, to make sure nothing breaks in master, before it's pulled into enterprise.

Reply:
What do you mean? Avi does not want the implementation of those verbs to be in OSS. How can you test without implementation of those verbs.  Here, only the two verbs are reserved.

Reply:
Hmm, so I actually want a test to exercise the serialization of temporary_buffer.
The verbs implementations are indeed out of scope.

Reply:
We have a unit test which uses a strong hash to make sure the data send over the rpc stream using temporary_buffer is identical on sender and receiver.